src/Mailchimp/Synchronisation/RequestBuilder.php

```diff
@@ -0,0 +1,251 @@
+        ;
+    }
+
+    public function setEmail(string $email): self
+    {
+        $this->email = $email;
+
+        return $this;
+    }
+
+    public function setGender(?string $gender): self
+    {
+        $this->gender = $gender;
+
+        return $this;
+    }
+
+    public function setFirstName(?string $firstName): self
+    {
+        $this->firstName = $firstName;
+
+        return $this;
+    }
+
+    public function setLastName(?string $lastName): self
+    {
+        $this->lastName = $lastName;
+
+        return $this;
+    }
+
+    public function setBirthDay(?\DateTime $birthDay): self
+    {
+        $this->birthDay = $birthDay;
+
+        return $this;
+    }
+
+    public function setCity(?string $city): self
+    {
+        $this->city = $city;
+
+        return $this;
+    }
+
+    public function setZipCode(?string $zipCode): self
+    {
+        $this->zipCode = $zipCode;
+
+        return $this;
+    }
+
+    public function setInterests(array $interests): self
+    {
+        $this->interests = $interests;
+
+        return $this;
+    }
+
+    public function setActiveTags(array $activeTags): self
+    {
+        $this->activeTags = $activeTags;
+
+        return $this;
+    }
+
+    public function setInactiveTags(array $inactiveTags): self
+    {
+        $this->inactiveTags = $inactiveTags;
+
+        return $this;
+    }
+
+    public function setCommitteeFollower(bool $committeeFollower): self
+    {
+        $this->committeeFollower = $committeeFollower;
+
+        return $this;
+    }
+
+    public function setCommitteeHost(bool $committeeHost): self
+    {
+        $this->committeeHost = $committeeHost;
+
+        return $this;
+    }
+
+    public function setCommitteeSupervisor(bool $committeeSupervisor): self
+    {
+        $this->committeeSupervisor = $committeeSupervisor;
+
+        return $this;
+    }
+
+    public function setCitizenProjectHost(bool $citizenProjectHost): self
+    {
+        $this->citizenProjectHost = $citizenProjectHost;
+
+        return $this;
+    }
+
+    public function buildMemberRequest(string $memberIdentifier): MemberRequest
+    {
+        $request = new MemberRequest($memberIdentifier);
+
+        if ($this->email) {
+            $request->setEmailAddress($this->email);
+        }
+
+        $request->setMergeFields($this->buildMergeFields());
+
+        if ($this->interests) {
+            $request->setInterests($this->interests);
+        }
+
+        return $request;
+    }
+
+    public function createMemberTagsRequest(string $memberIdentifier, array $removedTags = []): MemberTagsRequest
+    {
+        $request = new MemberTagsRequest($memberIdentifier);
+
+        foreach (array_merge($removedTags, $this->inactiveTags) as $tagName) {
+            $request->addTag($tagName, false);
+        }
+
+        foreach ($this->activeTags as $tagName) {
+            $request->addTag($tagName, true);
```

Comment:
`public function addTag(string $tagName, bool $active = true)`
active id true by default, was it intentional ?